lib/core/network/api/services/token_service.dart

```diff
@@ -96,12 +97,12 @@ class TokenService {
     required String symbol,
   }) async {
     try {
-      final requestBody = '''
-      { 
-        "json": true,
-        "code": "$tokenContract",
-        "symbol": "$symbol"
-      }''';
+      final requestBody = {
+        'json': true,
+        'code': tokenContract,
+        'symbol': symbol
+      };
+
```

Comment:
Did you test these?

I guess the library makes proper json strings out of these... ?! the API rejects it if there's not " around all keys and values ... but that is the normal JSON spec so I'd expect it to work.

Reply:
yes i did.